build.gradle

```
@@ -23,6 +23,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.boot:spring-boot-starter-validation'
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.11.0'
```

스프링의 [버전 관리 문서](https://docs.spring.io/spring-boot/docs/3.0.x/reference/html/dependency-versions.html#appendix.dependency-versions)를 보면 gson에 대한 부분의 버전 관리를 해주고 있으므로 버전 정보는 빼주셔도 될 것 같습니다!
```suggestion
    implementation 'com.google.code.gson:gson'
```